docs/content/docs/concepts/signals.md

```diff
@@ -6,6 +6,92 @@ discussion: 5469
 aliases:
   - "/docs/advanced/signals/"
   - "/docs/deep/signals/"
+draft: true
 ---
 
-Signals are similar to ops, but are not stored; they're useful to implement features like user presence.
+When using DDSes, data is sequenced and stored within the Fluid container to achieve synchronized shared state. For scenarios that involve shared persisted data, DDSes provide an effective way to communicate data so that it is retained in the container. However, there are many scenarios where we need to data communicate that is short-lived, in which the ordering and storage of said information would be wasteful and unnecessary.
+
+In contrast, the data that is communicated via signals is not retained in the container, which makes it an appropriate channel for transmitting transient data. A good analogy to help conceptualize signals is that they can be interpreted as UDP protocol, whereas standard ops can be thought of as TCP protocol. Signals are not guaranteed to be ordered on delivery relative to other signals and ops, but they still provide a useful communication channel for impermanent and short-lived data.
+
+## Why are signals useful?
+Signals provide a communication channel for sharing short-lived information that does not need to be persisted in the Fluid op stream.
```

Comment:
Have we decided that we're going to use "op" as a term or "operation"? If the former, is there a definition of "op" that we can link to?

Reply:
Yeah, Matt mentioned something similar [here](https://github.com/microsoft/FluidFramework/pull/11153#discussion_r923860523). I think replacing "Fluid op stream" with "Fluid container" would be better.

Reply:
> Have we decided that we're going to use "op" as a term or "operation"? If the former, is there a definition of "op" that we can link to?

Don't think we have a definition for op in FF.com. Don't remember seeing anything. Good item to add to our the future work list.